upstairs/src/lib.rs

```diff
@@ -5567,6 +5561,11 @@ impl Upstairs {
 
         Ok(notify_guest)
     }
+
+    async fn ds_repair_address(&self, client_id: u8, addr: SocketAddr) {
```

Comment:
This name, maybe call it `ds_set_repair`? Or something to indicate this is setting the repair address?

It is a little confusing that there is a `Downstairs` `repair_addr` method that will return the repair address for a given client ID, and an `Upstairs` method (this here) which will set the repair address.

Reply:
done in 2585e35